### PR TITLE
Mental health Existing Conditions conditional rendering

### PR DESCRIPTION
**Pull request by floresj**
Issue #954 
- Renders Existing Conditions menu item when all other mental subsections have top level branch values of 'No'
- `Diagnoses` section next button renders Review or Existing Conditions depending on the values of the other mental health sections
- Updated tests

### CODE REVIEW

File: src/config/navigation.js [hunk 1]
```diff
@@ -121,7 +123,7 @@ const navigation = [
       { name: 'Consultations', url: 'consultations' },
       { name: 'Hospitalizations', url: 'hospitalizations' },
       { name: 'Diagnoses', url: 'diagnoses' },
-      { name: 'Existing Conditions', url: 'conditions' }
+      { name: 'Existing Conditions', url: 'conditions', hiddenFunc: hideExistingConditions }
```
Comment (bmallred) [inline src/config/navigation.js]: :nauseated_face: 

I'm not a big fan but it's probably the best way to handle this scenario

Comment (floresj) [inline src/config/navigation.js]: We could look into making the navigation declarative like the `SectionView` stuff so we can add in clearer hooks into manipulating how items are rendered.

Comment (bmallred) [inline src/config/navigation.js]: My reasoning of dislike is simply add logic to a configuration file. It's good as-is but I would make a mental note to re-evaluate at a later date. Right now this is only planned on being used for one use case.

Comment (floresj) [inline src/config/navigation.js]: :+1:

**Pull request merged**